### DISCUSSION TITLE
Duplicates on feeds & explore

### DISCUSSION THREAD

**Discussion by OlivierJM** [community/community#68194] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

As many have already been complaining lately, the new dashboard feed is the worst updates Github has pushed in a long time, I used to rely on the home page to see things that were most important to me for that particular day but now it's either I see too much more than what i need or I don't see anything at all. 

Screenshots below show an example of how useless it can at times, The first one shows duplicates in explore page, the second one shows my feed nothing there that interests me or that I care about, why dont I just see activity from people i follow or repos I starred 

![Screenshot 2023-09-25 at 9 59 37 AM](https://github.com/community/community/assets/11255454/5cf4b841-58c0-48be-92df-0cb670695cf7)

![Screenshot 2023-09-19 at 10 54 15 AM copy](https://github.com/community/community/assets/11255454/735bbfe3-ac33-46ab-9d4f-a453ae1d639b)

Another one 

![Screenshot 2023-09-25 at 6 07 07 PM](https://github.com/community/community/assets/11255454/5195479d-6374-4e43-b119-f83b001495db)

**Comment by deltabeard** [comment]
I just got the same too. The feed is becoming more and more useless. Also, I've previously starred that repository, so I don't see why it should be in my "explore" list as I've already explored it!

![Screenshot 2023-09-25 at 11-30-15 Build software better together](https://github.com/community/community/assets/3747104/311f497f-d624-4f2e-812c-79aafec60e00)

**Comment by qurious-pixel** [comment]
Please add the ability to stack. 
Would rather have one stack than the many repeating releases for this (and other) repos.

![stack_repos](https://github.com/community/community/assets/62252937/36081580-c9b0-4bf8-9b96-dfe94502f956)

**Comment by shihabmridha** [comment]
Same here.
![image](https://github.com/community/community/assets/3264156/ff785a92-8160-40f1-8c03-2822081301e4)

**Comment by selimdoyranli** [comment]
![image](https://github.com/community/community/assets/22690563/964ea0d7-60d8-4b2d-abdd-c5cc184516be)

**Comment by bwkam** [comment]
Same here, even with the same exact repos.
I don't even see the "Explore Repositories" section anymore.

**Comment by MarvinJWendt** [comment]
I just went here to report this. Exact same repo:
![image](https://github.com/community/community/assets/31022056/474a350d-57f0-44fe-8734-b35ed463d8e8)

**Comment by ghosind** [comment]
![WX20230926-091955](https://github.com/community/community/assets/5114584/2befd901-863c-4a2b-ad77-10a1717c9f7e)
I also got same here, and always these repos

**Comment by bwkam** [comment]
![image](https://github.com/community/community/assets/91009118/7076c75a-fc3d-4acd-a861-7074210d80dc)
Getting those again.

**Comment by devoren** [comment]
<img width="189" alt="image" src="https://github.com/community/community/assets/99968085/2f991016-338d-428f-8524-3415f680df06">

**Comment by AyakuraYuki** [comment]
![image](https://github.com/community/community/assets/8560583/1bd1a2ac-125d-4aed-838b-f3aeba52f5c1)

Great, triple valhalla!!!

**Comment by GeoZac** [comment]
![Screenshot 2023-10-09 172127](https://github.com/community/community/assets/11597134/b6ac1ade-3328-42a5-80c0-985bdf7e7bc9)
Well who's the other one? Both cards had the same user

**Comment by webdiscus** [comment]
![Screenshot 2023-10-10 at 10 36 28](https://github.com/community/community/assets/6752572/2bd8c6a1-0d68-464d-a7f1-1875e4c5cad5)

Duplicates of the same starred repository.
